HaloCEVR/WeaponHandler.cpp

```diff
@@ -880,3 +889,8 @@ void WeaponHandler::PostThrowGrenade(HaloID& playerID)
 		weaponFiredPlayer = nullptr;
 	}
 }
+
+Vector3 WeaponHandler::Lerp(const Vector3& a, const Vector3& b, float t) const
```

Comment:
I wonder if this would be good to move into one of the math files

Reply:
Agreed, this is now resolved.

Reply:
Updated files for testing: [HaloCEVR.zip](https://github.com/user-attachments/files/17982655/HaloCEVR.zip)